packages/QnAMaker/bin/qnamaker.js

```diff
@@ -49,6 +49,12 @@ async function runProgram() {
     }
     args = minimist(argvFragment);
 
+    if (args.prefix) {
+        const unhook_intercept = intercept(function(txt) {
```

Comment:
I don't understand what's going on here. I see an unhook_intercept being defined, but now used anywhere. 

Can you explain a bit more what's going on here?

Reply:
@cleemullins, this was copied from [intercept-stdout](https://www.npmjs.com/package/intercept-stdout) samples, it can be used to stop intercepting. We removed it since it's not used in the CLI tool; we want to intercept output during the entire lifecycle of the CLI process.